src/contractTest/java/uk/gov/hmcts/reform/wataskmanagementapi/provider/TaskManagementGetTaskBySearchForCompletablePactTest.java

```diff
@@ -107,36 +118,37 @@ public void getTasksBySearchForCompletableCriteriaWithWarnings() {
             "1617708245335311",
             "refusalOfHumanRights",
             "Bob Smith",
-            true,
-            new WarningValues(emptyList()),
-            "Some Case Management Category",
+            false,
+            new WarningValues(Collections.emptyList()),
+            "Case Management Category",
             "hearing_work",
-            new TaskPermissions(
-                new HashSet<>(
-                    asList(
-                        PermissionTypes.READ,
-                        PermissionTypes.OWN,
-                        PermissionTypes.EXECUTE,
-                        PermissionTypes.CANCEL,
-                        PermissionTypes.MANAGE,
-                        PermissionTypes.REFER
-                    ))),
+            permissions,
             RoleCategory.LEGAL_OPERATIONS.name(),
             "a description"
         );
-
         return singletonList(task);
     }
 
     public List<Task> createTasksWithWarnings() {
         final List<Warning> warnings = List.of(
             new Warning("Code1", "Text1")
         );
-        WarningValues warningValues = new WarningValues(warnings);
+
+        final TaskPermissions permissions = new TaskPermissions(
+            Set.of(
+                PermissionTypes.READ,
+                PermissionTypes.OWN,
+                PermissionTypes.EXECUTE,
+                PermissionTypes.CANCEL,
+                PermissionTypes.MANAGE,
+                PermissionTypes.REFER
+            )
+        );
+
         Task taskWithWarnings = new Task(
             "4d4b6fgh-c91f-433f-92ac-e456ae34f72a",
             "Review the appeal",
-            "reviewTheAppeal",
+            "ReviewTheAppeal",
```

Comment:
Same comment as before

And also this FT should not be delete:
https://github.com/hmcts/wa-task-management-api/blob/52f5167b776df841773cb6a9f5fa1ee9ac1ddd04/src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/watasks/controllers/PostTaskInitiateByIdControllerTest.java